Godeps/Godeps.json

```diff
@@ -391,43 +391,33 @@
 			"Rev": "52e4358cbd540cc31f72ea5e0bd4762c98011b84"
 		},
 		{
-			"ImportPath": "github.com/containernetworking/cni/pkg/types",
-			"Comment": "v0.3.0-83-g52e4358",
-			"Rev": "52e4358cbd540cc31f72ea5e0bd4762c98011b84"
-		},
-		{
-			"ImportPath": "github.com/containernetworking/cni/pkg/invoke",
-			"Comment": "v0.2.3",
-			"Rev": "52e4358cbd540cc31f72ea5e0bd4762c98011b84"
-		},
-		{
 			"ImportPath": "github.com/containernetworking/cni/pkg/ip",
-			"Comment": "v0.2.3",
+			"Comment": "v0.3.0-83-g52e4358",
```

Comment:
I didn't check the history of what it's out of sync, but it's definitely cni commit 52e4358.